contracts/token/common/ERC2981/IERC2981Internal.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.8;
+
+import { IERC2981 } from '../../../interfaces/IERC2981.sol';
+
+/**
+ * @title ERC2981 interface
+ */
+interface IERC2981Internal {
```

Comment:
Move this to `interfaces/` and inherit it into `IERC2981`.  All interfaces that don't have "modifiers" such as "Base" or the "SolidState" prefix go in `interfaces/`.